1001-2000/LIT1284Dersan.xml

```diff
@@ -42,97 +42,718 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+               </div>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamTahsas" corresp="LIT6822SalamLakaAqabi">
+                  <label xml:lang="en">Salām-hymn to you, keeper</label>
+                  <note>According to <bibl><ptr target="bm:GrebTiss1935Codices"/><citedRange unit="page">575</citedRange></bibl>, 
+                     the <foreign xml:lang="gez">salām</foreign> is identical to <ref type="work" corresp="LIT1295Dersan#SalamTahsas"/>, but it is dedicated to the Archangel
+                     Gabriel instead of Michael.</note>
+               </div>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="ConsecrationDana" corresp="LIT7717Consecra">
+                  <label xml:lang="en">On the consecration of the church of St Gabriel in Dānā.</label>
+                  <div type="textpart" subtype="incipit">
+                     <note>The incipit follows <ref type="mss" corresp="BAVet82"/>.</note> 
+                     <ab>
+                        በስመ፡ አብ፡ <gap reason="ellipsis"/> አመ፡ ፳ወ፪ለወርኃ፡ ታኅሣሥ፡ በዛቲ፡ ዕለት፤ ኮነ፡ ተዝካረ፡ በዓሉ፡ ለመልአክ፡ ክቡር፡ ወቅዱስ፡ ገብርኤል፡ ሊቀ፡
+                        መላእክት፡ መበስር፡ ወሕንፀተ፡ ቤተ፡ ክ<supplied reason="omitted">ር</supplied>ስቲያኑ፡ በውስተ፡ ሀገረ፡ ዳና፤ ወአርአየ፡ መልአክ፡ ተአምራቲሁ፡ ባቲ፡
+                        ወቅዳሴሃ፡ በከመ፡ ዛቲ፡ ዕለት፨ ወኮነ፡ ስምዓ፡ በዝንቱ፡ አርከላኦስ፡ ቀሲስ፡ ኤጲስ፡ ቆጶስ፡ ዘይእቲ፡ ሀገር፤
+                     </ab>
+                  </div>                  
+               </div>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Annunciation" corresp="LIT4035SenkessarDL#Tahs19Gabriel">
+                  <label xml:lang="en">On the institution of the feast of the Annunciation.</label>
+                  <note>According to <bibl><ptr target="bm:GrebTiss1935Codices"/><citedRange unit="page">575</citedRange></bibl>, 
+                     the note corresponds to <ref type="work" corresp="LIT4035SenkessarDL#Tahs19Gabriel"/>.</note>
+                  <div type="textpart" subtype="incipit">
+                     <note>The incipit follows <ref type="mss" corresp="BAVet149"/>.</note> 
+                     <ab>
+                        ወካዕበ፡ በዛቲ፡ ዕለት፡ <sic>ትዝካረ፡</sic> በዓል፡ ዓቢይ፡ በዘዜነዋ፡ ሊቀ፡ መላእክት፡ ገብርኤል፡ ወአብሰራ፡ ለእግዝእትነ፨ 
+                        ማርያም<supplied reason="omitted">፡</supplied> ወላዲተ፡ አምላክ፡ ወሀሎ፡ ፩ብእሲ፡ ኤጲስ፡ ቆጶስ፡ በሀገረ፡ ሮሜ፨ ዘስሙ፡ ደቅስዮስ፡
+                     </ab>
+                  </div>                  
+               </div>
             </div>
+            
+            
             <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Terr">
                <label>
-                  <foreign xml:lang="gez">Ṭǝrr</foreign>
+                  Reading for 19 Ṭǝrr
                </label>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyZechariah" corresp="LIT7718HomilyGab">
```

Comment:
Not relevant to the present PR but perhaps it would be useful to add a relation to differentiate between LIT7718HomilyGab and LIT1634Homily